### PR TITLE
Qrda

### PR DESCRIPTION
**Pull request by dczulada**
This adds support for QDM->QRDA export and QRDA->QDM import.  Roundtrip calculations using this library have been compared with the "expected" calculations.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR:
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass.  The only Rubocop error is for a "cop" that is no longer in use.

**Reviewer 1:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,14 +2,18 @@ source "https://rubygems.org"
 
 gemspec :development_group => :test
 
+gem 'mustache'
+gem 'cqm-models', :git => 'https://github.com/projecttacoma/cqm-models', :branch => 'results_object'
```
Comment (mayerm94) [inline Gemfile]: Will there be a PR for results_object to be in PR to master soon? Results_object and master have diverged quite a bit so this should be done sooner rather than later

File: lib/qrda-export/catI-r5/qrda_header/_custodian.mustache [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<custodian>
+  <assignedCustodian>
+    <representedCustodianOrganization>
+      <!-- HQR Only -->
+      <id extension="800890" root="2.16.840.1.113883.4.336"/>
+      <name>Cypress Test Deck</name>
```
Comment (mayerm94) [inline lib/qrda-export/catI-r5/qrda_header/_custodian.mustache]: Since this repo is intended to be used elsewhere this information should be parameterized at qrda-generation

File: lib/qrda-import/base-importers/demographics_importer.rb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+        # gender_node = patient_element.at_xpath('cda:administrativeGenderCode')
+        # patient.gender = gender_node['code']
+        # id_node = patient_role_element.at_xpath('./cda:id')
+        # patient.medical_record_number = id_node['extension']
+        
+        # parse race, ethnicity, and spoken language
+        # race_node = patient_element.at_xpath('cda:raceCode')
+        # patient.race = { 'code' => race_node['code'], 'codeSystem' => 'CDC Race' } if race_node
+        # ethnicity_node = patient_element.at_xpath('cda:ethnicGroupCode')
```
Comment (mayerm94) [inline lib/qrda-import/base-importers/demographics_importer.rb]: What is/was all of this commented code for?
